tests/integration/modules/molecule/pipe_handler/playbook.yml

```diff
@@ -0,0 +1,129 @@
+---
+- name: Converge
+  collections:
+    - sensu.sensu_go
+  hosts: all
+  gather_facts: true
+  tasks:
+    - name: Create pipe handler with minimal parameters
+      pipe_handler:
+        auth:
+          url: http://localhost:8080
+        name: minimal_handler
+        command: sensu-influxdb-handler
+      register: result
+
+    - assert:
+        that:
+          - result is changed
+          - result.object.metadata.name == 'minimal_handler'
+
+    - name: Create pipe handler with minimal parameters idempotence
+      pipe_handler:
+        auth:
+          url: http://localhost:8080
+        name: minimal_handler
+        command: sensu-influxdb-handler
+      register: result
+
+    - assert:
+        that: result is not changed
+
+    - name: Create an pipe handler
+      pipe_handler:
+        auth:
+          url: http://localhost:8080
+        name: handler
+        command: sensu-influxdb-handler
+        mutator: mutate_input
+        timeout: 30
+        filters:
+          - occurances
+          - production
+        env_vars:
+          INFLUXDB_ADDR: http://influxdb.default.svc.cluster.local:8086
+          INFLUXDB_USER: sensu
+          INFLUXDB_PASS: password
+        runtime_assets:
+          - sensu-influxdb-handler
+      register: result
+
+    - assert:
+        that: result is changed
```

Comment:
I think we should add more assert statements to check all the parameters; unless there is some problem doing it I don't see?